x-pack/plugins/apm/public/components/app/transaction_details/waterfall_with_summary/waterfall_container/Waterfall/span_flyout/index.tsx

```diff
@@ -35,8 +35,9 @@ import { HttpInfoSummaryItem } from '../../../../../../shared/Summary/http_info_
 import { TimestampTooltip } from '../../../../../../shared/TimestampTooltip';
 import { ResponsiveFlyout } from '../ResponsiveFlyout';
 import { SyncBadge } from '../sync_badge';
-import { DatabaseContext } from './database_context';
+import { SpanDatabase } from './span_db';
```

Comment:
Renamed `DatabaseContext` to `SpanDatabase` since `span.context.db` was renamed to `span.db` long time ago.